spec/models/gift_exchange_spec.rb

```diff
@@ -1,18 +1,45 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe GiftExchange do
+  it do
```

Comment:
On purpose there's no description there?

Reply:
Yes; normally, I'd do something like `it { is_expected.to do_something }` (https://stackoverflow.com/a/25729231) with [Shoulda matchers](https://github.com/thoughtbot/shoulda-matchers). But in this case, it ended up being quite long. I can add a description if that would make things clearer, but Shoulda will output something fairly understandable without